libc-test/build.rs

```diff
@@ -1990,6 +1990,9 @@ fn test_freebsd(target: &str) {
     cfg.define("_WITH_GETLINE", None);
     // Required for making freebsd11_stat available in the headers
     cfg.define("_WANT_FREEBSD11_STAT", None);
+    cfg.define("LIBICONV_PLUG", None);
+
+    cfg.flag("-I/usr/local/include");
```

Comment:
Ideally, we should use the value of `sysctl -n user.localbase` here instead of hardcoding "/usr/local", but I don't know how to do it in Rust.